### PR TITLE
Document process for GitHub roadmap

### PR DESCRIPTION
**Pull request by jasonyavorsky**
This PR documents that there is a new OKR/roadmap tracker in GitHub, links to it and the corresponding slide deck, and documents the weekly update process around them.

- https://deploy-preview-699--sourcegraph-handbook.netlify.app/product/product_management#keeping-strategy-up-to-date

### CODE REVIEW

File: content/product/product_management/index.md [hunk 1]
```diff
@@ -20,15 +20,20 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
 
 In addition to this shared content, each team has a set of sources of truth (that do not overlap) that they should update regularly:
 
-#### Per-area strategy pages
+#### OKR and roadmap tracking
 
-We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) to communicate where each of our product teams is headed, for the areas of the product that they work on. Newly created teams should begin documenting the scope of their team as well as their mission, vision, principles, and more by creating one of these pages as part of the process of forming. Having a page like this will help you ensure your own framework for prioritization is clear, and can also be a resource for those outside your team who want to understand why you are working on what you are.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your engineering manager to update this content, which is the source of truth for goal and roadmap tracking.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your team to update this content, which is the source of truth for goal and roadmap tracking.
```

File: content/product/product_management/index.md [hunk 2]
```diff
@@ -20,15 +20,20 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
...
 
-These are updated monthly by each team, but wouldn't normally change too much each month. They are updated via a pull request, that anyone on the team can contribute to, and that can be shared with stakeholders as a summary of the change.
+After updating this content, you should ensure it is reflected in the manually created slide decks which rely on this information:
+
+- The [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]: If we have the content available in an internally available source of truth now, could the task of copying it to the PMM deck for marketing purposes be done by the marketing team instead of the product managers?

File: content/product/product_management/index.md [hunk 3]
```diff
@@ -20,15 +20,20 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
...
+- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]: Similar question - I don't think we answered who owns this. @christinaforney or @nicksnyder do you own copying the data from the GitHub dashboard into the key accounts deck on some cadence or do you expect product managers to do it when they make updates to the dashboard?

File: content/product/product_management/index.md [hunk 4]
```diff
@@ -20,15 +20,20 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
...
 
-#### PMM Roadmap slides
+#### Per-area strategy pages
+
+We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) to communicate where each of our product teams is headed beyond the near-term roadmap, for the areas of the product that they work on. Newly created teams should begin documenting the scope of their team as well as their mission, vision, principles, and more by creating one of these pages as part of the process of forming. Having a page like this will help you ensure your own framework for prioritization is clear, and can also be a resource for those outside your team who want to understand why you are working on what you are.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a framework for each team's prioritization, to document the trade-offs between goals the teams are making, and to communicate where they are headed in the longer term not in terms of specific deliverables but in terms of strategy.
```

File: content/product/product_management/index.md [hunk 5]
```diff
@@ -20,15 +20,20 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
...
+- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts.
 
-#### PMM Roadmap slides
+#### Per-area strategy pages
+
+We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a framework for each team's prioritization, to document the trade-offs between goals the teams are making, and to communicate where they are headed in the longer term not in terms of specific deliverables but in terms of strategy.
 
-In addition to updating the strategy pages monthly, at the same time the team should also update the [PMM roadmap](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming important customer-facing features.
+These are updated monthly by each team, but wouldn't normally change too much each month. They are updated via a pull request, that anyone on the team can contribute to, and that can be shared with stakeholders as a summary of the change.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
```

File: content/product/product_management/index.md [hunk 6]
```diff
@@ -20,15 +20,19 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
...
 
-These are updated monthly by each team, but wouldn't normally change too much each month. They are updated via a pull request, that anyone on the team can contribute to, and that can be shared with stakeholders as a summary of the change.
+- The [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker.
+- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts.
+
+#### Per-area strategy pages
 
-#### PMM Roadmap slides
+We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a framework for each team's prioritization, to document the trade-offs between goals the teams are making, and to communicate where they are headed in the longer term not in terms of specific deliverables but in terms of strategy.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a framework for each team's prioritization, to document the trade-offs between goals the teams are making, and to communicate where they are headed in the longer term not in terms of specific deliverables but in terms of strategy. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```

File: content/company/strategy/code-graph/index.md [hunk 7]
```diff
@@ -45,7 +45,7 @@ If we achieve these points, we can make it easy for developers to get things don
 
 We are focused on several themes to help make our one year vision a reality. They are not listed in priority order. To see how these play into what any particular product area is delivering, take a look at the individual team strategy pages linked at the [top of this page](#code-graph).
 
-We also set our goals around these themes, sou can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality.
+We also set our goals around these themes, sou can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality. The current status can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only).
```
Comment (jasonyavorsky) [inline content/company/strategy/code-graph/index.md]:
```suggestion
We also set our goals around these themes, sou can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality. The current status can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/11) (internal only).
```

File: content/product/product_management/index.md [hunk 8]
```diff
@@ -16,25 +16,24 @@ This page contains information that is relevant for how to do well at your job a
 
 ### Keeping strategy up to date
 
-Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents.
+Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents. In addition to this shared content, each team has a set of sources of truth (that do not overlap) that they should update regularly:
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents. In addition to this shared content, each team has a set of non-overlapping sources of truth that they should update regularly:
```

File: content/product/product_management/index.md [hunk 9]
```diff
@@ -16,25 +16,24 @@ This page contains information that is relevant for how to do well at your job a
 
 ### Keeping strategy up to date
 
-Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents.
+Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents. In addition to this shared content, each team has a set of non-overlapping sources of truth that they should update regularly:
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents. In addition to this shared content, each team has a set of non-overlapping sources of truth that they should update monthly:
```

File: content/product/product_management/index.md [hunk 10]
```diff
@@ -16,25 +16,24 @@ This page contains information that is relevant for how to do well at your job a
...
 
-In addition to this shared content, each team has a set of sources of truth (that do not overlap) that they should update regularly:
+#### OKR and roadmap tracking
 
-#### Per-area strategy pages
-
-We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) to communicate where each of our product teams is headed, for the areas of the product that they work on. Newly created teams should begin documenting the scope of their team as well as their mission, vision, principles, and more by creating one of these pages as part of the process of forming. Having a page like this will help you ensure your own framework for prioritization is clear, and can also be a resource for those outside your team who want to understand why you are working on what you are.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your team to update this content, which is the source of truth for goal and roadmap tracking. After updating this content, you should ensure it is reflected in the manually created slide decks which rely on this information:
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your team to update your plans and status. After updating, you should ensure it is reflected in the manually created slide decks which rely on this information:
```

File: content/product/product_management/index.md [hunk 11]
```diff
@@ -16,25 +16,24 @@ This page contains information that is relevant for how to do well at your job a
...
+- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts.
 
-#### PMM Roadmap slides
+#### Per-area strategy pages
 
-In addition to updating the strategy pages monthly, at the same time the team should also update the [PMM roadmap](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming important customer-facing features.
+We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a framework for each team's prioritization, to document the trade-offs between goals the teams are making, and to communicate where they are headed in the longer term not in terms of specific deliverables but in terms of strategy. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a narrative framework for prioritization, to document trade-offs between goals, and to communicate strategy outwardly. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```

File: content/product/product_management/index.md [hunk 12]
```diff
@@ -16,25 +16,24 @@ This page contains information that is relevant for how to do well at your job a
...
 
 #### Product data
 
 The last thing teams should review and update monthly are the product [data files](https://github.com/sourcegraph/handbook/blob/main/data/), in particular the [features.yml](https://github.com/sourcegraph/handbook/blob/main/data/features.yml) in case the feature has reached a new maturity, delivered new code host compatibility, or otherwise updated any information displayed on the automatically generated [feature reference pages](../index.md#feature-reference).
 
-Part of the product data is an internal-only list of [what's supported](https://docs.google.com/spreadsheets/d/101JXaau2EPvi322AOFmNeoeuXSJqlruD8gBBsHl1fmI/edit#gid=33376279), which contains extended product information for our Sales and Support teams. This should also be reviewed as feature capabilities evolve.
+Part of the product data is an internal-only list of [what's supported](https://docs.google.com/spreadsheets/d/101JXaau2EPvi322AOFmNeoeuXSJqlruD8gBBsHl1fmI/edit#gid=33376279), which contains extended product information for our Sales and Support teams. This should also be manually reviewed and updated as feature capabilities evolve.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
Part of the product data is a (internal-only) list of [what's supported](https://docs.google.com/spreadsheets/d/101JXaau2EPvi322AOFmNeoeuXSJqlruD8gBBsHl1fmI/edit#gid=33376279), which contains extended product information for our Sales and Support teams. This should also be manually reviewed and updated as feature capabilities evolve.
```

File: content/product/index.md [hunk 13]
```diff
@@ -13,9 +13,9 @@ You can reach us at the #product channel or @product-team on Slack. If you have
 
 ## Roadmap & Strategy
 
-The product strategy with how we think about prioritization and what we're working on next can be found on the [Sourcegraph strategy page](../company/strategy/index.md).
+The public product strategy with how we think about prioritization and what we're working on next can be found on the [Sourcegraph strategy page](../company/strategy/index.md).
```
Comment (jasonyavorsky) [inline content/product/index.md]:
```suggestion
The public strategy containing our mission, vision, how we think about prioritization, and what we're working on next can be found on the [Sourcegraph strategy page](../company/strategy/index.md).
```

File: content/product/product_management/index.md [hunk 14]
```diff
@@ -16,25 +16,24 @@ This page contains information that is relevant for how to do well at your job a
 
 ### Keeping strategy up to date
 
-Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents.
+Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents. In addition to this shared content, each team has a set of non-overlapping sources of truth that they should update monthly:
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) that describes at a high level where our product is headed and why. Everyone contributes to this page, and it's important to be familiar with its contents. In addition to this shared content, each team has a set of non-overlapping sources of truth that they should review and update monthly:
```

File: content/product/product_management/index.md [hunk 15]
```diff
@@ -16,25 +16,24 @@ This page contains information that is relevant for how to do well at your job a
...
+- The [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker.
+- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts.
 
-#### PMM Roadmap slides
+#### Per-area strategy pages
 
-In addition to updating the strategy pages monthly, at the same time the team should also update the [PMM roadmap](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming important customer-facing features.
+We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a narrative framework for prioritization, to document trade-offs between goals, and to communicate strategy outwardly. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a [narrative framework](https://www.mindtheproduct.com/the-importance-of-narrative/) for prioritization, to document trade-offs between goals, and to communicate strategy outwardly. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```

File: content/product/product_management/index.md [hunk 16]
```diff
@@ -20,11 +20,14 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
...
+- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts.
 
 #### Per-area strategy pages
 
-We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a narrative framework for prioritization, to document trade-offs between goals, and to communicate strategy outwardly. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
+We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a [narrative framework](https://www.mindtheproduct.com/the-importance-of-narrative/) for prioritization, to document trade-offs between goals, and to communicate strategy outwardly. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a narrative framework for prioritization, to document trade-offs between goals, and to communicate strategy outwardly. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```

File: content/company/strategy/code-graph/index.md [hunk 17]
```diff
@@ -45,7 +45,7 @@ If we achieve these points, we can make it easy for developers to get things don
 
 We are focused on several themes to help make our one year vision a reality. They are not listed in priority order. To see how these play into what any particular product area is delivering, take a look at the individual team strategy pages linked at the [top of this page](#code-graph).
 
-We also set our goals around these themes, sou can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality.
+We also set our goals around these themes, sou can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality. The current status can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/11) (internal only).
```
Comment (jasonyavorsky) [inline content/company/strategy/code-graph/index.md]:
```suggestion
We also set our goals around these themes, you can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality. The current status can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/11) (internal only).
```

File: content/product/product_management/index.md [hunk 18]
```diff
@@ -20,7 +20,10 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
 
 #### OKR and roadmap tracking
 
-Each month you should work with your team to update your plans. After updating, you should ensure it is reflected in the [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which is used by marketing to share with customers.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your team to update your plans. After updating, you should ensure it is reflected in the manually created slide decks which rely on this information:
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your team to update your status and plans. After updating, you should ensure it is reflected in the manually created slide decks which rely on this information:
```

File: content/company/strategy/code-graph/index.md [hunk 19]
```diff
@@ -45,7 +45,7 @@ If we achieve these points, we can make it easy for developers to get things don
 
 We are focused on several themes to help make our one year vision a reality. They are not listed in priority order. To see how these play into what any particular product area is delivering, take a look at the individual team strategy pages linked at the [top of this page](#code-graph).
 
-We also set our goals around these themes, you can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality.
+We also set our goals around these themes, you can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality. The current status can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/11) (internal only).
```
Comment (jasonyavorsky) [inline content/company/strategy/code-graph/index.md]:
```suggestion
We also set our goals around these themes, you can see current (Q4 2021) goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality. The current status can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/11) (internal only) or on the [slides version of the same content](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) (also internal only).
```

File: content/product/index.md [hunk 20]
```diff
@@ -15,7 +15,7 @@ You can reach us at the #product channel or @product-team on Slack. If you have
...
+Our roadmap (internal only) can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1), and a [manually updated slide version of the same content](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing). There is also a [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker, and a [key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) which contains status information on key accounts.
```
Comment (jasonyavorsky) [inline content/product/index.md]:
```suggestion
Our roadmap (internal only) can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1), and a [manually updated slide version of the same content](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing). There is also a [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker, and a [key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) which contains status information on key accounts.
```

File: content/company/strategy/code-graph/index.md [hunk 21]
```diff
@@ -45,7 +45,7 @@ If we achieve these points, we can make it easy for developers to get things don
...
 
-We also set our goals around these themes, you can see current FY22-Q4 goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality.
+We also set our goals around these themes, you can see current FY21-Q4 goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality. The current status can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/11) (internal only) or on the [slides version of the same content](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) (also internal only).
```
Comment (jasonyavorsky) [inline content/company/strategy/code-graph/index.md]: We are in Q4 of FY21, correct? FY22 starts in February of 2022?

Comment (jasonyavorsky) [reply content/company/strategy/code-graph/index.md]: Nope I'm wrong.

```suggestion
We also set our goals around these themes, you can see current FY22-Q4 goals as a bullet list in each theme. The team listed after each goal is the primary team responsible for implementing the goal, but in the end we achieve these together and will support each other as needed to make them reality. The current status can be found at our [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/11) (internal only) or on the [slides version of the same content](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) (also internal only).
```

File: content/product/product_management/index.md [hunk 22]
```diff
@@ -20,7 +20,11 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
 
 #### OKR and roadmap tracking
 
-Each month you should work with your team to update your plans. After updating, you should ensure it is reflected in the [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which is used by marketing to share with customers.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your team to update your status and plans. After updating, you should ensure it is reflected in the manually created slide decks which rely on this information:
+
+- The [OKR slide deck](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) which contains an easy to read version of the information in the tracker.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]: Confirmed in Slack that PMs are responsible for updating the OKR deck from the OKR tracker.

File: content/product/product_management/index.md (hunk 3 above)
Comment (jasonyavorsky) [reply content/product/product_management/index.md]: Maybe another related idea, could the OKR deck and key accounts deck be combined to reduce the number places to manually copy info to?

File: content/product/product_management/index.md [hunk 23]
```diff
@@ -16,11 +16,20 @@ This page contains information that is relevant for how to do well at your job a
...
 
 #### OKR and roadmap tracking
 
-Each month you should work with your team to update your plans. After updating, you should ensure it is reflected in the [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which is used by marketing to share with customers.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each week you should work with your team to update your status and plans. After updating, you should ensure it is reflected in the manually created slide decks which rely on this information:
+
+Weekly (by Friday 19:00 UTC):
+
+- The [OKR slide deck](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) which contains an easy to read version of the information in the tracker.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
- The [OKR slide deck](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) which contains an easy to read version of the information in the tracker. This document contains only OKRs at the product/engineering level, not for the orgs.
```

File: content/product/product_management/index.md [hunk 24]
```diff
@@ -16,11 +16,20 @@ This page contains information that is relevant for how to do well at your job a
...
 
 #### OKR and roadmap tracking
 
-Each month you should work with your team to update your plans. After updating, you should ensure it is reflected in the [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which is used by marketing to share with customers.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each week you should work with your team to update your status and plans. After updating, you should ensure it is reflected in the manually created slide decks which rely on this information:
+
+Weekly (by Friday 19:00 UTC):
+
+- The [OKR slide deck](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) which contains an easy to read version of the information in the tracker. This document contains only OKRs at the product/engineering level, not for the orgs.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
- The [OKR slide deck](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) which contains an easy to read version of the information in the tracker. This document contains only OKRs at the product/engineering level, not individual org OKRs.
```

File: content/product/product_management/index.md (hunk 2 above)
Comment (christinaforney) [reply content/product/product_management/index.md]: I think there is value in the product team making sure content is up to date and correct, but it does make sense for the PMM team to own.

File: content/product/product_management/index.md (hunk 3 above)
Comment (christinaforney) [reply content/product/product_management/index.md]: I own this (with @kyfli 's help). The key accounts deck contains the 5 slides about our key customers and is updated primarily by the CEs and AEs. The OKR deck didn't contain exact copied info, but pulls up the most important info to surface and track. I think they serve different purposes. 

I think the thing that I haven't solved for yet is what information is tracked in the GitHub board as the source of truth (vs. the key accounts deck). I'll circle around with Aimee and Gregg to see if we need to make any changes. 

TLDR; I own this, and the copying burden for this shouldn't fall on the PMs

**Review by christinaforney — APPROVED** [review]

File: content/product/product_management/index.md (hunk 3 above)
Comment (jasonyavorsky) [reply content/product/product_management/index.md]: Thanks @christinaforney !

```suggestion
- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts. Christina and Kylie own making this update for the product team.
```

File: content/product/product_management/index.md (hunk 2 above)
Comment (jasonyavorsky) [reply content/product/product_management/index.md]: We can leave as is for now then @christinaforney and see if we want to adjust in the future if it makes sense.

```suggestion
- The [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker. In the future we may transition updating this to PMM, but for now its important product managers are involved in ensuring it is up to date and correct.
```

File: content/product/product_management/index.md [hunk 25]
```diff
@@ -16,11 +16,20 @@ This page contains information that is relevant for how to do well at your job a
...
 
 #### OKR and roadmap tracking
 
-Each month you should work with your team to update your plans. After updating, you should ensure it is reflected in the [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which is used by marketing to share with customers.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each week you should work with your team to update your status and plans. After updating, you should ensure it is reflected in the manually created slide decks which rely on this information:
+
+Weekly (by Friday 19:00 UTC):
+
+- The [OKR slide deck](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) which contains an easy to read version of the information in the tracker. This document contains only OKRs at the product/engineering level, not individual org OKRs.
```
Comment (jasonyavorsky) [inline content/product/product_management/index.md]:
```suggestion
- The [OKR slide deck](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit?usp=sharing) which contains an easy to read version of the information in the tracker. This document contains only OKRs at the product/engineering level, not individual org OKRs, and is updated by Product Directors (based on the updates provided by product teams).
```

**Comment by jasonyavorsky** [comment]
Thanks so much for the feedback @christinaforney this is looking good!
